mlir/lib/Dialect/AIR/IR/AIRDialect.cpp

```diff
@@ -1027,6 +1027,7 @@ uint64_t HerdOp::getNumRows() {
 
 LogicalResult ExecuteOp::verify() {
   assert(getOperation()->getNumRegions() == 1 && "ExecuteOp has zero region!");
+  assert(!getRegion().empty() && "ExecuteOp should have non-empty region");
   assert(!getBody().empty() && "ExecuteOp should have non-empty body");
```

Comment:
These should `return 
```suggestion
  if (getOperation()->getNumRegions() != 1)
    return emitOpError("ExecuteOp has zero region!");
  if (getRegion().empty())
    return emitOpError("ExecuteOp should have non-empty region");
  if (getBody().empty())
    return emitOpError("ExecuteOp should have non-empty body");
```

Reply:
Make sense. Updated.